src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
-body {
+$cell_size : 100px;
+$start-day : ( 
+  "tue" : 1,
+  "wed" : 2,
+  "thu" : 3,
+  "fri" : 4,
+  "sat" : 5,
+  "sun" : 6
+);
+
+.html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
   margin: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  align-items: center;
+  width: $cell_size * 7 + 6;
+  text-align: center;
+
+  @each $day, $n in $start-day {
+    &--start-day-#{$day} &_cell:first-child {
+      margin-left: $n * ($cell-size + 1px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    &--#{$day} &_cell:nth-child(-n + #{$day}) {
+      display: flex;
+    }
+  }
```

Comment:
instead of set `display: none` by default set `display: flex` and set `display: none` in loop for the last days:
```
@for $index from 29 to 31 {
    &--#{$index} :nth-child(n + #{$index + 1}) {
      display: none;
    }
  }
```

Reply:
Every time i learn something new. I didnt know that i can #{$index + 1}. I wrote #{$index} + 1 and it didnt work correct. So thats why i used -n and display: flex.